docs/development_suite/company/create.mdx

```diff
@@ -7,228 +7,560 @@ sidebar_label: Create a Company
 
-Let's now configure the company to prepare itself to the creation of a new project.
+You will be able to set all the optional fields after the creation of the company.
+For further information regarding other configuration fields, visit the dedicated [section](#default-configuration-for-a-new-project) later in this page.
 
 #### Add the first Company User
 
-:::info
-If you used the API to create the company, you can skip this section.
+Once your company has been created using CMS, you need to associate the first user to be able to access the company from the Console. 
+The first user may either be yourself or another user in the Platform. 
+
+Regardless, head over to the CMS page and click the `Bindings` menu item.
+
+:::caution
+Only a restricted set of users have access to the users management section in CMS, ask your Console Administrator if you need the right permissions to access this functionality.
 :::
 
-If your company has been created by CMS, you have to add the first user to it, who may be yourself or another user in the Platform. Regardless, head over to the CMS page and choose the _Bindings_ menu item (only a restricted set of users have access to this section, ask your Console Administrator if you need access to it).
+In the `Bindings` section, click the `Add new` button and fill in the form:
+- type a custom **Binding ID**
 
-In the _Bindings_ section, click the _Create new_ button and fill in the form, type a custom _Binding ID_, select the proper Role (since it is the first user, usually it should be `Company Owner`) and make sure to select the correct user as _Subject_. Finally, you'll have to assign the correct resource by filling it with an object as follows:
+  :::danger
+  The _Binding ID_ must be unique in the whole collection!  
 
-```json
-{
-  "resourceType": "company",
-  "resourceId": "THE ID YOU PROVIDED UPON CREATION AS tenantId"
-}
-```
+  As a general recommendation we suggest `{companyId}-{desiredRole}`, however, in some cases a more complex structure may be necessary.
+  :::
 
-:::danger
-The _Binding ID_ must be unique in the whole collection!  
-As a general recommendation we suggest `{tenantId}-{desiredRole}`, but in some cases something more complex may be needed.
+- select the proper **Role** (since it is the first user, it is recommended to set the role [`Company Owner`](/docs/development_suite/identity-and-access-management/console-levels-and-permission-management#users-capabilities-inside-console).)
+
+- select the correct user as **Subject**
+
+- assign the correct **Resource** by filling it with an object as follows:
+
+  ```json
+  {
+    "resourceType": "company",
+    "resourceId": "{THE COMPANY ID YOU PROVIDED UPON CREATION}"
+  }
+  ```
+## Company Configuration
+
+Once you create a company, you can configure:
+
+- providers (git providers, secret managers, CI/CD tools)
```

Comment:
Dato che questo probabilmente è il primo accenno ai provider in un flusso utente spiegherei brevissimamente cosa intendiamo noi come providers e per cosa vengono utilizzati nella console. Magari un primo utente non ne hai idea e non collega leggendo un generico "provider"